### DISCUSSION TITLE
Certificates

### DISCUSSION THREAD

**Discussion by jbcjsc** [ImageMagick/ImageMagick#2753] [General]
When I try to install /any/ Win32-version of Github (I've tried current versions and legacy's), it always says "not a valid Win32-application" (or at least: the Dutch wording of this...) When I tried looking at the properties, I saw the RSA-certificate was valid until a week ago. So I suppose this might be the cause of the problem.
Does anyone recognize this?
Would it be possible to refresh the RSA-certificate?
Grtz.,
Jaap

**Comment by dlemstra** [comment]
What is the operating system that you are trying to install this on? And which file are you downloading from where?

The certificate has expired and we are in the process of renewing it. But that is unrelated to this issue because the signing of the application is done with a timestamp. This means that when we signed it the certificate was valid.

**Reply by jbcjsc** [reply]
Dear D. Lemstra,

Thank you for replying so soon.

I'm trying to install on an ancient (I admit), but genuine WinXP-version (that I run on VirtualBox 6.1 in Linux Mint).
I downloaded from https://imagemagick.org/script/download.php all the 7.0.10-34-versions that said "win32", both the installers and the portables. All the installers said the same: not a valid win32-executable. After unzipping the portables, I tried convert.exe, but I got the same message every time.
I also tried downloading from https://legacy.imagemagick.org/script/download.php, all the 6.9.34-versions that said "win32". Same results.

I normally keep my WinXP away from the internet (I have Virtual Box disconnect the network and transfer files from the internet using a shared folder). I thought that maybe internet were required to check on the certificate. Testing this theory, I connected my XP to the internet while trying the downloaded files, but to no avail.

I hope I gave you useful information. If you need any more info, please let me know. Your help is much appreciated.

Kind regards,

Jaap

**Reply by dlemstra** [reply]
We stopped support WindowsXP. So you will either need to use a more recent version of Windows or use a version that was still compatible with WindowsXP. You can find more info about that here: https://github.com/ImageMagick/ImageMagick/discussions/2376

**Reply by jbcjsc** [reply]
Dear Dirk,
I was under the impression that "Win32" would still include WinXP, but apparently, I am mistaken - it's not the first mistake in my life ;-) Anyway, that explains why I didn't look for threads on WinXP and support. Maybe I should have done that, it would have saved you time and trouble. So: apologies.
Anyway: the reference to the other thread proved most useful, I've been able to get me a working version.
Thank you very much for you swift and accurate reply!
All the best,
Jaap

**Reply by dlemstra** [reply]
No worries. Thanks for your friendly reply.
